src/libsystemd/sd-varlink/sd-varlink.c

```diff
@@ -3660,11 +3660,12 @@ _public_ int sd_varlink_server_add_connection_stdio(sd_varlink_server *s, sd_var
         return 0;
 }
 
-_public_ int sd_varlink_server_listen_auto(sd_varlink_server *s) {
+_public_ int sd_varlink_server_listen_name(sd_varlink_server *s, const char *name) {
         _cleanup_strv_free_ char **names = NULL;
         int r, n = 0;
 
         assert_return(s, -EINVAL);
+        assert_return(name, -EINVAL);
 
         /* Adds all passed fds marked as "varlink" to our varlink server. These fds can either refer to a
```

Comment:
Still not updated?

Reply:
???
I added the line as requested by you?

Reply:
The function no longer works on "varlink" fdname, but what's specified by caller. Hence the comment shouldn't state that anymore